test/tools/test_configure.py

```diff
@@ -0,0 +1,24 @@
+import sys, os, unittest, subprocess
+
+class ConfigureTests(unittest.TestCase):
+  def setUp(self):
+    self.working_dir = os.path.abspath(
+      os.path.join(
+        os.path.dirname(__file__),
+        '..', '..'
+      )
+    )
+
+  def test_ninja(self):
+    subprocess.check_call(
+      './configure --ninja',
+      cwd=self.working_dir,
+      shell=True,
+      stdout=subprocess.PIPE,
+      stderr=subprocess.PIPE
+    )
+
+if __name__ == '__main__' and\
+  sys.platform in ['linux', 'linux2', 'darwin', 'cygwin']:
```

Comment:
This is awesome work!  Thanks.  Line 21 should be a backslash, not a slash?  One minor nit.  PEP8 strongly discourages the use of backslashes in Python code because one whitespace to the right of the backslash breaks the script on a change that is invisible to the user.  Could we instead use parens:
```python
if (__name__ == '__main__' and
    sys.platform in ('linux', 'linux2', 'darwin', 'cygwin')):

Reply:
Updated and fixed a few other pep8 warnings.